portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,32 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Kevin Liaw</h1>
+      <img src="/images/face.jpg" alt="my photo" width="500" height="600">
       <p>This is my portfolio.</p>
+      <p>
+        Hello everyone, my name is Kevin Liaw. I'm a upcoming sophmore
+        at University of Washington. On my free time I play video games 
+        and watch Netflix.
+      </p>
       <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <button onclick="addRandomGreeting()">Movie Quote</button>
       <div id="greeting-container"></div>
     </div>
+
+    <div>
+        <h2>Hobbies:</h2>
+        <p>Swimming<p>
+        <img src="/images/lcat.gif" alt="logo of LCAT">
+        <p>
+            I learend how to swim on LCAT (Las Cruces Aquatic Team) since
+            I was six years old.  
+        <p>
+        <p></p>
+    </div>
+
+    <footer>
+        <p>github: https://github.com/kev14540</p>
+    </footer>
   </body>
-</html>
+</html>
```

Comment:
It's often considered good form to have a final newline at the end of the file. Probably largely historical but can be helpful when using command line tools like grep and sed.